django/applications/catmaid/static/js/widgets/stack-viewer-grid.js

```diff
@@ -369,6 +369,54 @@
     scaleBarCbLabel.appendChild(scaleBarCb);
     this.controlsContainer.appendChild(scaleBarCbLabel);
 
+    var zoomInput = document.createElement('input');
+    zoomInput.setAttribute('type', 'text');
+    zoomInput.setAttribute('pattern', '^\-?\d?\.?\d*$');
+    zoomInput.setAttribute('size', '5');
+    // zoomInput.setAttribute('type', 'number');
+    // zoomInput.setAttribute('step', '0.1');
+    // zoomInput.setAttribute('max', '5');
+    // zoomInput.setAttribute('min', '-2');
```

Comment:
If you don't think the number type should be used, this commented code should be removed.